### PR TITLE
Finish Removing snake_case from Drawing Methods

### PR DESCRIPTION
**Pull request by duncanspumpkin**
Thought this would take longer but no this is the last one. Might clash with #19167

### CODE REVIEW

File: src/openrct2-ui/drawing/engines/opengl/OpenGLDrawingEngine.cpp [hunk 1]
```diff
@@ -206,7 +206,7 @@ class OpenGLDrawingEngine final : public IDrawingEngine
     {
         _window = static_cast<SDL_Window*>(_uiContext->GetWindow());
         _bitsDPI.DrawingEngine = this;
-        lightfx_set_available(false);
+        LightfxSetAvailable(false);
```
Comment (Gymnasiast) [inline src/openrct2-ui/drawing/engines/opengl/OpenGLDrawingEngine.cpp:209]: I would write `lightfx` as `LightFX`.

**Review by Broxzier — APPROVED** [review]

File: src/openrct2/drawing/Drawing.String.cpp [hunk 2]
```diff
@@ -40,7 +40,7 @@ enum : uint32_t
     TEXT_DRAW_FLAG_NO_DRAW = 1u << 31
 };
 
-static int32_t ttf_get_string_width(std::string_view text, FontStyle fontStyle, bool noFormatting);
+static int32_t TtfGetStringWidth(std::string_view text, FontStyle fontStyle, bool noFormatting);
```
Comment (Gymnasiast) [inline src/openrct2/drawing/Drawing.String.cpp:43]: TTF

**Review by Gymnasiast — CHANGES_REQUESTED** [review]
Please address the line notes I left.

File: src/openrct2/libopenrct2.vcxproj [hunk 3]
```diff
@@ -211,6 +211,8 @@
     <ClInclude Include="drawing\Font.h" />
     <ClInclude Include="drawing\IDrawingContext.h" />
     <ClInclude Include="drawing\IDrawingEngine.h" />
+    <ClInclude Include="drawing\Image.h" />
```
Comment (Gymnasiast) [inline src/openrct2/libopenrct2.vcxproj:214]: ?

**Review by Gymnasiast — APPROVED** [review]

**Pull request merged**